Source/relay/TourGuide/Quests/8-bit Realm.ash

```diff
@@ -6,16 +6,21 @@ void Q8BitInit()
 
     // Set the state as "started" if you have the continuum transfunctioner.
     if (!state.started && $items[continuum transfunctioner].available_amount() > 0)
-        QuestStateParseMafiaQuestPropertyValue(state, "started");
+        state.started = true;
```

Comment:
we really need to write a readme for some of these, i really did not correctly parse how this dang function worked lol